app/src/main/java/org/simple/clinic/summary/bloodpressures/newbpsummary/NewBloodPressureSummaryViewUi.kt

```diff
@@ -0,0 +1,5 @@
+package org.simple.clinic.summary.bloodpressures.newbpsummary
+
+interface NewBloodPressureSummaryViewUi {
```

Comment:
The reason why I added `New` prefix is because of `NewBloodPressureSummaryView`. I wanted all the files part of the `NewBloodPressureSummaryView` to be similarly named and hoping to rename them once rest of the views in PatientSummary is done. Should I rename them in this PR?

Reply:
This has been resolved in the next PR https://github.com/simpledotorg/simple-android/pull/1083